README.md

```diff
@@ -124,6 +124,115 @@ systemctl restart snap.shadowsocks-rust.sslocal-daemon.service
 # ... and show service status
 systemctl status snap.shadowsocks-rust.sslocal-daemon.service
 ```
+## A step-by-step snap install guide
+
+### ▍1. Intro
+I have been using SS-Libev(server) on my Google VPS for a long time. Since SS-Libev was discontinued and future development had moved to SS-Rust, thought I'd give it a try. 
+
+During deployment of SS-Rust with snap, like many issues posted, accessing config.json wasn't straightforward as SS-Libev. After spending a long time troubleshooting and finally made it work, I thought I could share my experience, and hence this guide.
+
+Here is a step-by-step guide to deploy Shadowsocks-Rust for newbies like me. See you on the free Internet!  
+- ​this guide was performed on `Ubuntu 20.04 LTS`. It should also work on other linux images with minor modification of the codes.
+___
+### ▍2. Environment Configuration
+
+2.1 (Optional) Obtain Root Privileges
+```sh
+# (Optional, as I have included "sudo" in almost all of the codes)
+sudo -i  # elevate to root user
+```
+
+2.2 Install Core Dependencies
+```sh
+# update apt
+sudo apt update && sudo apt upgrade -y
+# install snap package manager  
+sudo apt install snapd -y
+# install core snap components         
+sudo snap install core            
+```
+___
+### ▍3. Service Installation
+
+3.1 Install Shadowsocks-RUST
+```sh
+sudo snap install shadowsocks-rust
+```
+
+==**3.2 IMPORTANT**==: start shadowsocks-rust-ssserver-daemon before writing the config.json file**
+```sh
+snap start --enable shadowsocks-rust.ssserver-daemon
+# alternativety, "sudo systemctl start snap.shadowsocks-rust.ssserver-daemon" can also be used to initiate the service.
+# note: unlike SS-libev, in SS-RUST, the config.json file is somehow not created/inaccessible before ssserver-daemon starts. Hence, ssserver-daemon should be initiated before modifying config.json. Furthermore, it is recommended to use the full service name (i.e. snap.shadowsocks-rust.ssserver-daemon).
```

Comment:
You don't need to mention the libev version. The guide is just for this project.